Documentation/observability/metrics.rst

```diff
@@ -691,6 +691,30 @@ Name                                           Labels
 ``ces_queueing_delay_seconds``                                                  CiliumEndpointSlice queueing delay in seconds
 ============================================== ================================ ========================================================
 
+Identity GC
+~~~~~~~~~~~
+
+======================================== ============================== ======== ========================================================
+Name                                     Labels                         Default  Description
+======================================== ============================== ======== ========================================================
+``identity_gc_entries``                  ``status``, ``identity_type``  Enabled  The number of alive and deleted identities at the end of a garbage collector run
+``identity_gc_runs``                     ``outcome``, ``identity_type`` Enabled  The number of times identity garbage collector has run
+======================================== ============================== ======== ========================================================
+
+"Double Write" Identity Allocation Mode
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+When the ":ref:`Double Write <double_write_migration>`" identity allocation mode is
+enabled, the following metrics are available:
+
+================================ ======= ========== ============================================================
+Name                             Labels  Default    Description
+================================ ======= ========== ============================================================
+``identity_crd_total_count``             Disabled   The total number of CRD identities
+``identity_kvstore_total_count``         Disabled   The total number of identities in the KVStore
+``identity_crd_only_count``              Disabled   The number of CRD identities not present in the KVStore
+``identity_kvstore_only_count``          Disabled   The number of identities in the KVStore not present as a CRD
```

Comment:
The default should be `Enabled` here, in the sense that the metrics are enabled by default (you didn't specify `Disabled: true` as part of the definition. Then they are obviously present only if the allocation mode is doublewrite.